### PR TITLE
Display Publication Date and Collection Dates in 'YYYY-MM-DD' format;…

### PR DESCRIPTION
**Pull request by coblej**
… RDR-228

### CODE REVIEW

**Comment by dchandekstark** [comment]
@mcdowney @willsexton @susivey Are you sure you want to facet on the day (as opposed to, say, the year)?

File: app/views/hyrax/datasets/_dataset_versions.html.erb [hunk 1]
```diff
@@ -33,7 +33,9 @@
 	<%= Array(ver.provenance).last %>
       </td>
       <td>
-	<%= Array(ver.available).first %>
+  <% if Array(ver.available).first %>
+    <%= Date.parse(Array(ver.available).first).to_formatted_s(:standard) %>
```
Comment (dchandekstark) [inline app/views/hyrax/datasets/_dataset_versions.html.erb]: @coblej Seems that we can use the `readable_date` helper?

File: app/renderers/faceted_date_attribute_renderer.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  def li_value(value)
+    link_to(ERB::Util.h(Date.parse(value).to_formatted_s(:standard)), search_path(value))
+  end
+
+  def search_path(value)
+    Rails.application.routes.url_helpers.search_catalog_path(:"f[#{search_field}][]" => value)
+  end
+
+  def search_field
+    ERB::Util.h(Solrizer.solr_name(options.fetch(:search_field, field), :dateable, type: :string))
```
Comment (dchandekstark) [inline app/renderers/faceted_date_attribute_renderer.rb]: @coblej Let's look at this together.

**Comment by coblej** [comment]
@dchandekstark Revised PR ready for your (or someone's) re-review.  Thanks for the input!

**Pull request merged**